pkgs/tools/system/netdata/default.nix

```
@@ -60,18 +60,12 @@ stdenv.mkDerivation rec {
     ++ lib.optionals withSsl [ openssl ];
 
   patches = [
+    # Allow ndsudo to use non-hardcoded `PATH`
+    # See https://github.com/netdata/netdata/pull/17377#issuecomment-2183017868
+    #     https://github.com/netdata/netdata/security/advisories/GHSA-pmhq-4cxq-wj93
+    ./patches/ndsudo-fix-path.patch
     # Allow building without non-free v2 dashboard.
-    (fetchpatch {
-      url = "https://github.com/netdata/netdata/pull/17240/commits/b108df72281633234b731b223d99ec99f1d36adf.patch";
-      hash = "sha256-tgsnbNY0pxFU3bz1J1qPaAeVsozsk2bpHV2mNy8A9is=";
-    })
-    # Allow for go.d plugins to access the right directory.
-    # Can be removed once > v1.45.4 is released
-    # https://github.com/netdata/netdata/pull/17661
-    (fetchpatch {
-      url = "https://patch-diff.githubusercontent.com/raw/netdata/netdata/pull/17661.patch";
-      sha256 = "sha256-j+mrwkibQio2KO8UnV7sxzCoHmkcsalHNzP+YvrRz74=";
-    })
+    ./patches/dashboard-v2-removal.patch
```

We don't place patches in a subdirectory unless there are many.

```suggestion
    ./dashboard-v2-removal.patch
```